src/west/main.py

```diff
@@ -519,6 +520,14 @@ def main(argv=None):
         sys.exit(cce.returncode)
     except CommandError as ce:
         sys.exit(ce.returncode)
+    except BadExternalCommand:
+        log.err('external command', args.command,
+                'was improperly defined and could not be run')
```

Comment:
are we able to provide more info to the user, on how the command was improperly defined ?

Reply:
For now that is all we have. I think since this patch is an improvement and we are short on time we should discuss further improvements later.

Reply:
(Developers who are trying to debug can use `-v` to get the stack trace  -- users won't care, they just know "it didn't work")

Reply:
thanks.

Reply:
I've added a patch which does all of this. You convinced me :)

Reply:
I just asked out of curiosity, and was satisfied with your answer.
Appears you are easy to convince :)